Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ set(_SAINT_NEC_ENGINE_LIB_SRC
 	"physics/PhysicObject.cpp"
 	"physics/util.cpp"
 	"graphics/TessellatedPlane.cpp"
+	"architecture/PhysicManager.cpp"
+	"physics/algorithms.cpp"
```

Comment:
Lower case first letter

Reply:
As it is not a class but only function shouldn't we keep lower case like for 'util' ?